osm2streets-js/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "osm2streets-npm"
```

Comment:
Is there a reason the package name is different from the directory name?

Reply:
Oops, fixed!